packages/app/hooks/use-sign-in.ts

```diff
@@ -0,0 +1,22 @@
+import { SignInParam } from 'app/hooks/use-sign-in-param'
```

Comment:
I don't think we need this hook, as we are using it only in one place and it's quite simple.